web/pages/[username]/[contractSlug].tsx

```diff
@@ -75,9 +88,10 @@ export async function getStaticPropz(props: {
   return {
     props: {
       contract,
-      bets,
+      bets: bets.slice(0, 100),
```

Comment:
I think the client assumes ascending order of bets? You could reverse this.

Or, change the client to assume descending order of bets! (i.e. listen for bets created after the first)

Reply:
just reversed it, thanks!